src/pools/index.ts

```diff
@@ -10,6 +10,7 @@ import { Gyro3Pool } from './gyro3Pool/gyro3Pool';
 import { GyroEPool } from './gyroEPool/gyroEPool';
 import { GyroEV2Pool } from './gyroEV2Pool/gyroEV2Pool';
 import { FxPool } from './xaveFxPool/fxPool';
+import { KassandraManagedPool } from './managedPools/kassandraManagedPool';
```

Comment:
Do you mind updating the pool name to `MaganedPoolKassandra`? I believe it will be easier to manage in case we have several different managed pools within the SOR.

Reply:
done